client/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "client",
```

Comment:
Would this not be Fakebook?

EDIT: So there is a Fakebook `package.json` down below. Which is the client folder? Why does this have a `package.json` too?

EDIT2: Okay, I get it now. This is the `react` front end SPA. The other `package.json` is for the `express` server. Perhaps this `package.json` should have more of the fakebook details in it (e.g. `fakebook-app`) and the other file have more server oriented info (e.g. `fakebook-api`)? Also should the server files be in a `server` folder? Currently it's a bit hard to figure out what's going on at a glance.

Reply:
I think I'd potentially have the two apps run seperately (i.e. not have anything about the client in the servers `package.json` and have the user start them both.

What you have is good though. No need for any more changes.